src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetInstanceWithPropertiesV46.sql

```diff
@@ -0,0 +1,57 @@
+﻿/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetInstanceWithPropertiesV46
+--
+-- FIRST SCHEMA VERSION
+--     32
+--
+-- DESCRIPTION
+--     Gets valid dicom instances at study/series/instance level with additional instance properties
+--
+-- PARAMETERS
+--     @invalidStatus
+--         * Filter criteria to search only valid instances
+--     @partitionKey
+--         * The Partition key
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetInstanceWithPropertiesV46 (
```

Comment:
Could you upload the SQL plan, this is currently used by WADO. Wanted to ensure we don't regress the current perf numbers